packages/perseus-editor/src/__stories__/interactive-graph-editor.stories.tsx

```diff
@@ -171,7 +171,18 @@ MafsWithLockedFiguresCurrent.parameters = {
 
 export const MafsWithLockedLabelsFlag = (): React.ReactElement => {
     return (
-        <EditorPageWithStorybookPreview question={segmentWithLockedFigures} />
+        <EditorPageWithStorybookPreview
+            apiOptions={{
+                flags: {
+                    mafs: {
+                        ...flags.mafs,
+                        "interactive-graph-locked-features-labels": true,
+                        "locked-point-labels": false,
```

Comment:
Is `locked-point-labels` supposed to be `false` here?

Reply:
Yes, because it's for the story with general labels but not point labels. Just for some flag testing.